### PR TITLE
Set up some boilerplate structures to represent our songs and songbooks. 

### PR DESCRIPTION
**Pull request by brotatotes**
Pretty incomplete but wanted to get this in instead of continuing to hold on to it. Set up some data structures to define songs/songbooks. Also played around with http get request to retrieve remote files. Still need logic to properly get metadata files and jpg files.

### CODE REVIEW

File: src/Songs.tsx
Comment (brandonxia01) [inline src/Songs.tsx:86]: nit: new line at the end of this file

File: resources/Songs_&_Hymns_Of_Life/metadata/434 - Just Taste And See That The Lord Is Good.json
Comment (brandonxia01) [inline resources/Songs_&_Hymns_Of_Life/metadata/434 - Just Taste And See That The Lord Is Good.json:2]: assuming this was a just test file?

File: resources/Songs_&_Hymns_Of_Life/metadata/418 - Faith Of Our Fathers Living Still.json
Comment (brandonxia01) [inline resources/Songs_&_Hymns_Of_Life/metadata/418 - Faith Of Our Fathers Living Still.json:1]: And this one?

**Review by brandonxia01 — APPROVED** [review]
Nice dude 👍

File: resources/Songs_&_Hymns_Of_Life/metadata/434 - Just Taste And See That The Lord Is Good.json
Comment (brotatotes) [reply resources/Songs_&_Hymns_Of_Life/metadata/434 - Just Taste And See That The Lord Is Good.json:2]: Actually I think it was a duplicate, so I deleted.

File: resources/Songs_&_Hymns_Of_Life/metadata/418 - Faith Of Our Fathers Living Still.json
Comment (brotatotes) [reply resources/Songs_&_Hymns_Of_Life/metadata/418 - Faith Of Our Fathers Living Still.json:1]: Duplicate, deleted.

**Review by brandonxia01 — APPROVED** [review]
Oh hmm if the PR changes then it needs re-approval, is that a setting we can change?

**Comment by brandonxia01** [comment]
I think I updated the setting - after approval, if someone commits after an approval is given, the approval will stay valid. LMK if we don't actually want that. @123asian - fyi

can you try to do another dummy commit on this PR just to confirm behavior

**Pull request merged**
